guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -56,6 +56,13 @@ These changes are applied to the `/etc/foreman-maintain/foreman-maintain-hammer.
 +
 Review the results and address any highlighted error conditions before performing the upgrade.
 
+. Optional: If you haven't migrated all the organizations to Simple Content Access, auto-migrate the organizations while upgrading:
```

Comment:
You can drop the words "while upgrading". This is an upgrading procedure that users always follow while upgrading.

Reply:
@asteflova thanks! Also, this command is to be executed if the orgs are not migrated manually earlier. I am wondering if adding the command to the prerequisites makes sense? 

@jeremylenz Will it make sense to add the command to the prerequisites, or is it to be executed strictly during the procedure?

Reply:
I don't want to encourage users to wait until the moment of upgrade to migrate their organizations. Rather, I want users to be fully informed and upgrade to SCA long before they get to this decision point.

If they've ignored all the warnings up until now, and they still attempt to upgrade, the pre-upgrade check will fail. The command needed to override it and continue anyway is in the output of foreman-maintain when the pre-upgrade check fails. I think that documenting it also here is not needed, and would encourage a behavior we don't want to encourage (namely, just blindly skipping the pre-upgrade check so you can continue with upgrading.)

Reply:
In other words, I suggest deleting this change, because it's already covered by

```
Review the results and address any highlighted error conditions before performing the upgrade.
```

Reply:
Thanks @jeremylenz! I have deleted the line from the procedure. Should we keep the migration to SCA in the prerequisites?

Reply:
Yes, I've got no issues with that as a prerequisite. But remember you'll have to update the docs in the next version to remove it.